packages/plugin-client-kysely/README.md

```diff
@@ -0,0 +1,33 @@
+# @xata.io/kysely
+
+A [Kysely](https://github.com/kysely-org/kysely) dialect for [Xata](https://xata.io), using the [Xata serverless driver for TypeScript](https://github.com/xataio/client-ts).
```

Comment:
cc: @joan-ing 

This shows up inside npm, do we want to link to our docs. 

What is going on with the kysely docs, who will be writing them?

Reply:
We will have our own docs, this readme will be for direct links from Kysely